src/dctkit/apps/linear_elasticity.py

```diff
@@ -8,9 +14,44 @@ def __init__(self, S: SimplicialComplex, mu_: float, lambda_: float):
         self.mu_ = mu_
         self.lambda_ = lambda_
 
-    def linear_elasticity_residual(self, g):
+    def __get_current_metric_2D(self, node_coords: npt.Array | Array):
```

Comment:
This should be a member of the SimplicialComplex class. The typing for numpy arrays is npt.NDArray